### PR TITLE
Added feature tests

### PR DESCRIPTION
**Pull request by kedves**
- Orchestra setup in order to write HTTP tests
- Added Account and Settings test

### CODE REVIEW

File: tests/TestUser.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Konekt\AppShell\Tests;
+
+use Konekt\AppShell\Models\User;
+
+class TestUser extends User
+{
+    protected $guard_name = 'web';
```
Comment (fulopattila122) [inline tests/TestUser.php]: This looks like faking the result, as there's no such requirement that user should have this property defined.
We need a solution without this. If you want it merged, please at least mark it as todo, push it to this branch and fix this hack in your next PR.

File: tests/TestCase.php [hunk 2]
```diff
@@ -39,38 +50,11 @@ protected function getPackageProviders($app)
...
     }
 
-    /**
-     * Set up the environment.
-     *
-     * @param \Illuminate\Foundation\Application $app
-     */
     protected function getEnvironmentSetUp($app)
     {
-        $app['config']->set('database.default', 'sqlite');
```
Comment (fulopattila122) [inline tests/TestCase.php:49]: Where will the test environment know from how to connect to the database?

File: tests/TestCase.php [hunk 3]
```diff
@@ -39,38 +50,11 @@ protected function getPackageProviders($app)
...
-    protected function setUpDatabase($app)
-    {
-        $app['db']->connection()->getSchemaBuilder()->create('users', function (Blueprint $table) {
-            $table->increments('id');
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->string('password');
-            $table->rememberToken();
-            $table->timestamps();
-        });
```
Comment (fulopattila122) [inline tests/TestCase.php:71]: Why was this removed? Does it work with all the supported Laravel versions?

File: phpunit.xml [hunk 4]
```diff
@@ -9,4 +9,9 @@
             <directory suffix=".php">./tests</directory>
         </testsuite>
     </testsuites>
+    <php>
+        <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
+        <env name="DB_CONNECTION" value="testing"/>
+        <env name="APP_DEBUG" value="true"/>
```
Comment (fulopattila122) [inline phpunit.xml]: Is it mandatory to run the tests in debug mode?

**Review by fulopattila122 — CHANGES_REQUESTED** [review]

File: phpunit.xml (hunk 4 above)
Comment (kedves) [reply phpunit.xml]: True. Isn't mandatory.

File: tests/TestCase.php (hunk 3 above)
Comment (kedves) [reply tests/TestCase.php:71]: In the `setUp()`  I'm calling `$this->loadLaravelMigrations()` which loads the laravels's out of the box migrations. It's part of the orchestra test library.

**Review by fulopattila122 — APPROVED** [review]
